docusaurus/docs/overview/feature-model/names.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Names
+---
+
+import JSONSchemaViewer from "@theme/JSONSchemaViewer";
+import generateResolverOptions from "@site/src/components/shared-libs/generateResolverOptions"
+import yamlLoad from "@site/src/components/yamlLoad"
+import MainDefs from "!!raw-loader!@site/docs/_schema/defs.yaml";
+
+## Common names schema
+
+Named entities in Overture share a common name schema.
+
+### `primary`
+
+If a feature has a name, expect the `names` attribute to have at least one property: `primary`. This will always be populated with the primary, localized name of a feature. Example: "Empire State Building" or "Eiffel Tower"
+
+```yaml
+names:
+  primary: The White House
+```
+
+
+### Variations
+
+#### `common`
+Common name variations are translations of a name.
+
+```yaml
+names:
+  common:
+    es: La Casa Blanca
+```
+
+#### `rules`
+Additional variations such as _official_, _alternate_, or _short_ name variations exist as `rules`.
```

Comment:
1. [ ] Ideally we should explain the link between `rules` and "Scoped and Rule-Based Properties".
2. [ ] Ideally we should give some further explanation of how to build up names rules and what your options are, *e.g.* language, side, LR, *etc.*
3. [ ] I think that the "rule evaluation algorithm" described on the scoped and rule-based properties page may need to be massaged to explain how to resolve name rules. I'm not 100% sure yet, but we should think about it.